error_test.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'environment.rb'
+
+class ErrorTest
+    include Environment
+    attr_accessor :board, :depot, :control, :dictionary, :poker
+  
+    def initialize
+      @board = { players: Hash.new, control: Hash.new, status: nil }
```

Comment:
Use `{} `instead `Hash.new`

Reply:
May I ask why {} is recommended?

Reply:
Sure, you are calling a constructor and no passing any parameters. 
Check this example in the  rubocop style guide
```
# bad
arr = Array.new
hash = Hash.new

# good
arr = []
arr = Array.new(10)
hash = {}
hash = Hash.new(0)
```
[Style-Guide](https://github.com/rubocop/ruby-style-guide#literal-array-hash)